Documentation/tutorial/intro_toc.md

```diff
@@ -0,0 +1,99 @@
+Table-Of-Content(TOC) Files
+===========================
+
+Introduction
+---------------------
+DocFX is able to process YAML files, which we call *Metadata File*s, and Markdown files, which we call *Conceptual File*s. Besides that, DocFX introduces a way to organize these files using *Table-Of-Content File*s, which we also call *TOC File*s, so that users can navigate through *Metadata File*s and *Conceptual File*s.
+
+*TOC File*s must have file name `toc.md` or `toc.yml`, notice that file name is case-insensitive.
+
+Basic format
+----------------
+### Markdown format TOC `toc.md`
+
+`toc.md` leverage Markdown [Atx-style headers](http://daringfireball.net/projects/markdown/syntax#header) which use 1-6 hash characters at the start of the line to represent the TOC levels 1-6. We call each line starting with hash characters a *TOC Item*. A *TOC Item* with higher level is considered as the child of the nearest upper *TOC Item* with less level. A sample `toc.md` is as below:
```

Comment:
inner vs. outer?

---
In reply to: [57005600](https://github.com/dotnet/docfx/pull/232#discussion_r57005600) [](ancestors = 57005600)